vscode/src/chat/chat-view/ChatController.ts

```diff
@@ -1136,7 +1131,7 @@ export class ChatController implements vscode.Disposable, vscode.WebviewViewProv
     }
 
     private async handleAbort(): Promise<void> {
```

Comment:
Should we update the signatures of the `cancelSubmitOrEditOperation` consumers too? 
```suggestion
    private handleAbort(): void {
```